dbt/include/trino/macros/catalog.sql

```diff
@@ -1,5 +1,6 @@
 {% macro trino__get_catalog(information_schema, schemas) -%}
     {%- call statement('catalog', fetch_result=True) -%}
+        {% set database = information_schema.database %}
```

Comment:
Please inline this expression in the query itself

Reply:
There is only one query in file. Where should I move it?

Reply:
Just do it in line 51 (or line 50 in original file) catalog_name = '{{ information_schema.database }}'